lib/resources/host.rb

```diff
@@ -93,6 +95,30 @@ def initialize(inspec)
     end
   end
 
+  class DarwinHostProvider < HostProvider
+    def ping(hostname, _port = nil, _proto = nil)
```

Comment:
Prefixing a parameter name with an underscore is generally used to indicate that the parameter is not going to be used. Now that we're using it, we should change `_proto` to `proto`, and `_port` to `port`. This is a Rubocop check, and appears to be the reason why the Travis tests are failing for your change, as well. 🙂